generate_outputs_gpt2.py

```diff
@@ -0,0 +1,41 @@
+from datasets import ClassLabel, load_dataset, Value 
+from transformers import AutoTokenizer, GPT2LMHeadModel
+import torch
+import argparse
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+
+if __name__=='__main__':
+    parser = argparse.ArgumentParser(description='Specify output path.')
+    parser.add_argument("--output_dir", type=str, required=True, help="Directory of the model.")
+    parser.add_argument("--checkpoint", type=str, required=True, help="Model checkpoint.")
+    args = parser.parse_args()
+
+
+    tokenizer = AutoTokenizer.from_pretrained(args.output_dir)
+    tokenizer.pad_token = tokenizer.eos_token
+
+    model = GPT2LMHeadModel.from_pretrained(args.output_dir+'/'+args.checkpoint).to(device)
+    f = open('test_mrs.txt', 'r+')
+    ls = f.readlines()
+    f.close()
```

Comment:
Use with for all file operations:
```suggestion
    with open('test_mrs.txt', 'r+') as f:
       ls = f.readlines()
```
Moreover, make this a shared function `load_input_data`